### DISCUSSION TITLE
Notifications for content that has missed scheduled publication date

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1282] [Feature Requests]
Here's some background:
https://www.wpkube.com/how-to-fix-the-missing-schedule-error-in-wordpress/

Here's the problem as seen by one user:

![Posts_--_Apartment_Therapy_--_WordPress-(1)](https://user-images.githubusercontent.com/3868028/57253458-c3ed1300-701c-11e9-864e-1e1310c89c12.jpg)

**Comment by stevejburge** [comment]
This plugin offers a possible fix 
https://wordpress.org/plugins/wp-missed-schedule-posts/

**Comment by andergmartins** [comment]
Today I received another request for this.
